app/router.js

```diff
@@ -105,6 +110,15 @@ module.exports = app => {
   app.get(`${offenderRoute}/edit-plan/new`, createSentencePlan)
   app.get(editPlanRoute, editPlan)
 
+  // edit contact arrangements
+  app.get(`${activePlanRoute}/contact-arrangements`, getContactArrangements)
+  app.post(
+    `${activePlanRoute}/contact-arrangements`,
```

Comment:
Duplication `${activePlanRoute}/contact-arrangements` but I'll merge and we can clean it up in housekeeping.